docs/controllers.md

```diff
@@ -44,8 +44,8 @@ implements the `ForwardingAttributes` interface.
 For implementation details, refer to the
 [proto definition](https://github.com/noble-assets/orbiter/blob/main/proto/noble/orbiter/core/v1/orbiter.proto#L35-L55).
 
-Forwarding controllers interpret protocol-specific information and forward funds.
-Based on the selected bridge, the following controllers are available:
+Forwarding controllers interpret protocol-specific information and forward funds. Based on the
+selected bridge, the following controllers are available:
```

Comment:
would be nice if the auto-prettier could be configured to use semantic line breaks 😅